### DISCUSSION TITLE
Unable to find some devices on Arduino Nano ESP32

### DISCUSSION THREAD

**Discussion by chinswain** [h2zero/NimBLE-Arduino#697] [General]
Has anyone used the Arduino Nano ESP32 with NimBLE?

I have a large number of LYWSD03MMC devices flashed with https://github.com/pvvx/ATC_MiThermometer inside beehives. I'm unable to detect them when scanning when using the custom firmware (detected with stock firmware), works fine on all other ESP32 dev board variants I have.

I don't think it's an issue with ESP32-S3 as it's working on an XIAO ESP32S3 Sense.

Details here: https://github.com/matthias-bs/ATC_MiThermometer/issues/13

**Comment by h2zero** [comment]
Interesting, no idea why this wouldn't work with that board. Hopefully someone has one that can answer.
